projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt

```diff
@@ -155,6 +158,7 @@ class ProjectorServer private constructor(
 
     builder.onWsClose = { connection ->
       // todo: we need more informative message, add parameters to this method inside the superclass
+      clientsCount = maxOf(0, clientsCount - 1)
```

Comment:
Strictly speaking, modifications of this property aren't thread-safe now. They should be synchronized or atomic